### PR TITLE
Fix Colony Info Command (for use with player names) 

### LINKED ISSUE #1259: [1.12] MC RTP does not check for world border and kills players.
### Minecolonies version

version X.X.XXXX or curse X.X.XX

### Expected behavior

- RTP checks against world border before sending player beyond it.

### Actual behaviour

- RTP drops people beyond the world border leading to instant death.

### Steps to reproduce the problem

1. Set world border to smaller then the rtp range.
2. Player uses rtp
3. Profit

### PR DESCRIPTION
**Pull request by Asherslab**
- Closes #1259 

# Changes proposed in this pull request:
- Fixes /col colony info <PLayerName> to be able to use offline players too. 
- Fixes RTP with world borders

Review please

### CODE REVIEW

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 28 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 26 minor

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [ShowColonyInfoCommand.java#L58](https://github.com/Minecolonies/minecolonies/blob/ab4cb720b250f1c88fbf488f9e5d6855a7bb3b41/src/main/java/com/minecolonies/coremod/commands/colonycommands/ShowColonyInfoCommand.java#L58): The Cyclomatic Complexity of this method "execute" is 16 which is greater than 15 authorized. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
1. ![MAJOR][MAJOR] [ShowColonyInfoCommand.java#L58](https://github.com/Minecolonies/minecolonies/blob/ab4cb720b250f1c88fbf488f9e5d6855a7bb3b41/src/main/java/com/minecolonies/coremod/commands/colonycommands/ShowColonyInfoCommand.java#L58): Refactor this method to reduce its Cognitive Complexity from 24 to the 20 allowed. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/ab4cb720b250f1c88fbf488f9e5d6855a7bb3b41/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/ab4cb720b250f1c88fbf488f9e5d6855a7bb3b41/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/ab4cb720b250f1c88fbf488f9e5d6855a7bb3b41/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [colony](https://github.com/Minecolonies/minecolonies/blob/ab4cb720b250f1c88fbf488f9e5d6855a7bb3b41/src/main/java/com/minecolonies/coremod/colony): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [buildings](https://github.com/Minecolonies/minecolonies/blob/ab4cb720b250f1c88fbf488f9e5d6855a7bb3b41/src/main/java/com/minecolonies/coremod/colony/buildings): Add a 'package-info.java' file to document the 'buildings' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [permissions](https://github.com/Minecolonies/minecolonies/blob/ab4cb720b250f1c88fbf488f9e5d6855a7bb3b41/src/main/java/com/minecolonies/coremod/colony/permissions): Add a 'package-info.java' file to document the 'permissions' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [creativetab](https://github.com/Minecolonies/minecolonies/blob/ab4cb720b250f1c88fbf488f9e5d6855a7bb3b41/src/main/java/com/minecolonies/coremod/creativetab): Add a 'package-info.java' file to document the 'creativetab' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [entity](https://github.com/Minecolonies/minecolonies/blob/ab4cb720b250f1c88fbf488f9e5d6855a7bb3b41/src/main/java/com/minecolonies/coremod/entity): Add a 'package-info.java' file to document the 'entity' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/java/com/minecolonies/coremod/commands/colonycommands/ShowColonyInfoCommand.java [hunk 1]
```diff
@@ -62,14 +63,22 @@ public void execute(@NotNull final MinecraftServer server, @NotNull final IComma
...
         if(colonyId == -1 && args.length >= 1)
         {
-            final EntityPlayer player = server.getEntityWorld().getPlayerEntityByName(args[0]);
+            final GameProfile playerProfile =  server.getPlayerProfileCache().getGameProfileForUsername(args[0]);
+
+            EntityPlayer player = null;
+
+            if (playerProfile != null)
+            {
+                player = (EntityPlayer) server.getEntityFromUuid(playerProfile.getId());
```
Comment (Raycoms) [inline src/main/java/com/minecolonies/coremod/commands/colonycommands/ShowColonyInfoCommand.java]: Have you tested that?
I bet that will only work for online players as well.

Comment (Raycoms) [reply src/main/java/com/minecolonies/coremod/commands/colonycommands/ShowColonyInfoCommand.java]: You won't be able to get the player entity. Check in the other command where we use it to get the colony owner with the game profile.

File: src/main/java/com/minecolonies/coremod/commands/colonycommands/ChangeColonyOwnerCommand.java [hunk 2]
```diff
@@ -51,7 +51,7 @@ public String getCommandUsage(@NotNull final ICommandSender sender)
     public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
```
Comment (MinecoloniesSonar) [inline src/main/java/com/minecolonies/coremod/commands/colonycommands/ChangeColonyOwnerCommand.java:51]: MAJOR Reduce the number of returns of this method 9, down to the maximum allowed 6. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1142)

File: src/main/java/com/minecolonies/coremod/commands/citizencommands/AbstractCitizensCommands.java [hunk 3]
```diff
@@ -44,24 +44,24 @@ public String getCommandUsage(@NotNull final ICommandSender sender)
     @Override
     public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
```
Comment (MinecoloniesSonar) [inline src/main/java/com/minecolonies/coremod/commands/citizencommands/AbstractCitizensCommands.java:45]: MAJOR Refactor this method to reduce its Cognitive Complexity from 21 to the 20 allowed. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 31 issues
* ![MAJOR][MAJOR] 5 major
* ![MINOR][MINOR] 26 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AbstractSingleCommand.java#L116](https://github.com/Minecolonies/minecolonies/blob/6a6fbfd287fa4325b5c09efbe2cdb9efbdd96fa0/src/main/java/com/minecolonies/coremod/commands/AbstractSingleCommand.java#L116): Reduce the number of returns of this method 13, down to the maximum allowed 6. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1142)
1. ![MAJOR][MAJOR] [ShowColonyInfoCommand.java#L58](https://github.com/Minecolonies/minecolonies/blob/6a6fbfd287fa4325b5c09efbe2cdb9efbdd96fa0/src/main/java/com/minecolonies/coremod/commands/colonycommands/ShowColonyInfoCommand.java#L58): Refactor this method to reduce its Cognitive Complexity from 22 to the 20 allowed. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)
1. ![MAJOR][MAJOR] [RandomTeleportCommand.java#L132](https://github.com/Minecolonies/minecolonies/blob/6a6fbfd287fa4325b5c09efbe2cdb9efbdd96fa0/src/main/java/com/minecolonies/coremod/commands/generalcommands/RandomTeleportCommand.java#L132): Reduce the total number of break and continue statements in this loop to use at most one. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS135)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/6a6fbfd287fa4325b5c09efbe2cdb9efbdd96fa0/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/6a6fbfd287fa4325b5c09efbe2cdb9efbdd96fa0/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/6a6fbfd287fa4325b5c09efbe2cdb9efbdd96fa0/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [colony](https://github.com/Minecolonies/minecolonies/blob/6a6fbfd287fa4325b5c09efbe2cdb9efbdd96fa0/src/main/java/com/minecolonies/coremod/colony): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [buildings](https://github.com/Minecolonies/minecolonies/blob/6a6fbfd287fa4325b5c09efbe2cdb9efbdd96fa0/src/main/java/com/minecolonies/coremod/colony/buildings): Add a 'package-info.java' file to document the 'buildings' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [permissions](https://github.com/Minecolonies/minecolonies/blob/6a6fbfd287fa4325b5c09efbe2cdb9efbdd96fa0/src/main/java/com/minecolonies/coremod/colony/permissions): Add a 'package-info.java' file to document the 'permissions' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [creativetab](https://github.com/Minecolonies/minecolonies/blob/6a6fbfd287fa4325b5c09efbe2cdb9efbdd96fa0/src/main/java/com/minecolonies/coremod/creativetab): Add a 'package-info.java' file to document the 'creativetab' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/java/com/minecolonies/coremod/commands/colonycommands/ShowColonyInfoCommand.java [hunk 4]
```diff
@@ -60,24 +61,25 @@ public void execute(@NotNull final MinecraftServer server, @NotNull final IComma
...
+
+            if (playerProfile != null)
             {
-                tempColony = ColonyManager.getIColonyByOwner(server.getEntityWorld(), player);
+                tempColony = ColonyManager.getIColonyByOwner(server.getEntityWorld(), playerProfile.getId());
             }
         }
 
-        if(sender instanceof EntityPlayer)
+        if (sender instanceof EntityPlayer && sender.getCommandSenderEntity() != null)
```
Comment (Raycoms) [inline src/main/java/com/minecolonies/coremod/commands/colonycommands/ShowColonyInfoCommand.java]: Already executed th enull check, no need.

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 31 issues
* ![MAJOR][MAJOR] 5 major
* ![MINOR][MINOR] 26 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AbstractSingleCommand.java#L116](https://github.com/Minecolonies/minecolonies/blob/879cddc9fed7d7be5eb2e0a5e63d5307ccdef18f/src/main/java/com/minecolonies/coremod/commands/AbstractSingleCommand.java#L116): Reduce the number of returns of this method 13, down to the maximum allowed 6. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1142)
1. ![MAJOR][MAJOR] [ShowColonyInfoCommand.java#L58](https://github.com/Minecolonies/minecolonies/blob/879cddc9fed7d7be5eb2e0a5e63d5307ccdef18f/src/main/java/com/minecolonies/coremod/commands/colonycommands/ShowColonyInfoCommand.java#L58): Refactor this method to reduce its Cognitive Complexity from 22 to the 20 allowed. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)
1. ![MAJOR][MAJOR] [RandomTeleportCommand.java#L132](https://github.com/Minecolonies/minecolonies/blob/879cddc9fed7d7be5eb2e0a5e63d5307ccdef18f/src/main/java/com/minecolonies/coremod/commands/generalcommands/RandomTeleportCommand.java#L132): Reduce the total number of break and continue statements in this loop to use at most one. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS135)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/879cddc9fed7d7be5eb2e0a5e63d5307ccdef18f/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/879cddc9fed7d7be5eb2e0a5e63d5307ccdef18f/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/879cddc9fed7d7be5eb2e0a5e63d5307ccdef18f/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [colony](https://github.com/Minecolonies/minecolonies/blob/879cddc9fed7d7be5eb2e0a5e63d5307ccdef18f/src/main/java/com/minecolonies/coremod/colony): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [buildings](https://github.com/Minecolonies/minecolonies/blob/879cddc9fed7d7be5eb2e0a5e63d5307ccdef18f/src/main/java/com/minecolonies/coremod/colony/buildings): Add a 'package-info.java' file to document the 'buildings' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [permissions](https://github.com/Minecolonies/minecolonies/blob/879cddc9fed7d7be5eb2e0a5e63d5307ccdef18f/src/main/java/com/minecolonies/coremod/colony/permissions): Add a 'package-info.java' file to document the 'permissions' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [creativetab](https://github.com/Minecolonies/minecolonies/blob/879cddc9fed7d7be5eb2e0a5e63d5307ccdef18f/src/main/java/com/minecolonies/coremod/creativetab): Add a 'package-info.java' file to document the 'creativetab' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 31 issues
* ![MAJOR][MAJOR] 5 major
* ![MINOR][MINOR] 26 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AbstractSingleCommand.java#L116](https://github.com/Minecolonies/minecolonies/blob/306ec2256f8ab8c32a1dfe39d3e37e57d62b2214/src/main/java/com/minecolonies/coremod/commands/AbstractSingleCommand.java#L116): Reduce the number of returns of this method 13, down to the maximum allowed 6. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1142)
1. ![MAJOR][MAJOR] [ShowColonyInfoCommand.java#L58](https://github.com/Minecolonies/minecolonies/blob/306ec2256f8ab8c32a1dfe39d3e37e57d62b2214/src/main/java/com/minecolonies/coremod/commands/colonycommands/ShowColonyInfoCommand.java#L58): Refactor this method to reduce its Cognitive Complexity from 21 to the 20 allowed. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)
1. ![MAJOR][MAJOR] [RandomTeleportCommand.java#L132](https://github.com/Minecolonies/minecolonies/blob/306ec2256f8ab8c32a1dfe39d3e37e57d62b2214/src/main/java/com/minecolonies/coremod/commands/generalcommands/RandomTeleportCommand.java#L132): Reduce the total number of break and continue statements in this loop to use at most one. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS135)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/306ec2256f8ab8c32a1dfe39d3e37e57d62b2214/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/306ec2256f8ab8c32a1dfe39d3e37e57d62b2214/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/306ec2256f8ab8c32a1dfe39d3e37e57d62b2214/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [colony](https://github.com/Minecolonies/minecolonies/blob/306ec2256f8ab8c32a1dfe39d3e37e57d62b2214/src/main/java/com/minecolonies/coremod/colony): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [buildings](https://github.com/Minecolonies/minecolonies/blob/306ec2256f8ab8c32a1dfe39d3e37e57d62b2214/src/main/java/com/minecolonies/coremod/colony/buildings): Add a 'package-info.java' file to document the 'buildings' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [permissions](https://github.com/Minecolonies/minecolonies/blob/306ec2256f8ab8c32a1dfe39d3e37e57d62b2214/src/main/java/com/minecolonies/coremod/colony/permissions): Add a 'package-info.java' file to document the 'permissions' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [creativetab](https://github.com/Minecolonies/minecolonies/blob/306ec2256f8ab8c32a1dfe39d3e37e57d62b2214/src/main/java/com/minecolonies/coremod/creativetab): Add a 'package-info.java' file to document the 'creativetab' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by marchermans — APPROVED** [review]

**Comment by Raycoms** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/1265/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 31 issues
* ![MAJOR][MAJOR] 5 major
* ![MINOR][MINOR] 26 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AbstractSingleCommand.java#L116](https://github.com/Minecolonies/minecolonies/blob/bcb21dd8c494c48300111364a1d46500890d8aec/src/main/java/com/minecolonies/coremod/commands/AbstractSingleCommand.java#L116): Reduce the number of returns of this method 13, down to the maximum allowed 6. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1142)
1. ![MAJOR][MAJOR] [ShowColonyInfoCommand.java#L58](https://github.com/Minecolonies/minecolonies/blob/bcb21dd8c494c48300111364a1d46500890d8aec/src/main/java/com/minecolonies/coremod/commands/colonycommands/ShowColonyInfoCommand.java#L58): Refactor this method to reduce its Cognitive Complexity from 21 to the 20 allowed. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)
1. ![MAJOR][MAJOR] [RandomTeleportCommand.java#L132](https://github.com/Minecolonies/minecolonies/blob/bcb21dd8c494c48300111364a1d46500890d8aec/src/main/java/com/minecolonies/coremod/commands/generalcommands/RandomTeleportCommand.java#L132): Reduce the total number of break and continue statements in this loop to use at most one. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS135)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/bcb21dd8c494c48300111364a1d46500890d8aec/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/bcb21dd8c494c48300111364a1d46500890d8aec/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/bcb21dd8c494c48300111364a1d46500890d8aec/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [colony](https://github.com/Minecolonies/minecolonies/blob/bcb21dd8c494c48300111364a1d46500890d8aec/src/main/java/com/minecolonies/coremod/colony): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [buildings](https://github.com/Minecolonies/minecolonies/blob/bcb21dd8c494c48300111364a1d46500890d8aec/src/main/java/com/minecolonies/coremod/colony/buildings): Add a 'package-info.java' file to document the 'buildings' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [permissions](https://github.com/Minecolonies/minecolonies/blob/bcb21dd8c494c48300111364a1d46500890d8aec/src/main/java/com/minecolonies/coremod/colony/permissions): Add a 'package-info.java' file to document the 'permissions' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [creativetab](https://github.com/Minecolonies/minecolonies/blob/bcb21dd8c494c48300111364a1d46500890d8aec/src/main/java/com/minecolonies/coremod/creativetab): Add a 'package-info.java' file to document the 'creativetab' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**